### ISSUE TITLE
pa11y 5.x beta error for empty HTML lang attribute

### ISSUE THREAD

**Issue by fdesjardins** [pa11y/pa11y#373]
While testing the beta version, we discovered having an empty `lang` attribute causes an error. For example, for
```html
<!doctype html>
<html lang="">
</html>
```
we get the debug info and error message
```
npx pa11y http://localhost:5001/ -d

Welcome to Pa11y

 > Running Pa11y on URL http://localhost:5001/
 > Debug: Launching Headless Chrome
 > Debug: Opening URL in Headless Chrome
 > Debug: Setting request method
 > Debug: Setting request headers
 > Debug: Injecting HTML CodeSniffer
 > Debug: Injecting Pa11y
 > Debug: Running Pa11y on the page
 > Debug: Document title: ""

Error: TypeError: Cannot read property 'replace' of null
    at report.issue.issue (/home/forrest/Workspace/github/project/web/node_modules/pa11y-reporter-cli/lib/reporter.js:54:40)
    at Array.map (<anonymous>)
    at report.results.results (/home/forrest/Workspace/github/project/web/node_modules/pa11y-reporter-cli/lib/reporter.js:34:21)
    at Object.results (/home/forrest/Workspace/github/project/web/node_modules/pa11y/lib/reporter.js:37:24)
    at runProgram (/home/forrest/Workspace/github/project/web/node_modules/pa11y/bin/pa11y.js:114:16)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7)
```

This error goes away if we either remove the attribute entirely or provide a value for it.

**Comment by rowanmanning** [comment]
Interesting, thanks for bringing to our attention. I'll take a look

**Comment by rowanmanning** [comment]
OK all fixed in #374. This should get merged and released tomorrow some time, thanks again for reporting 🙂

**Comment by rowanmanning** [comment]
Cool, fixed in `5.0.0-beta.9`
